package.json

```diff
@@ -9,6 +9,7 @@
   "private": true,
   "scripts": {
     "build": "tsc",
+    "postbuild": "rm -rf lib/test/ && rsync -a lib/src/ lib/ && rm -rf lib/src/",
```

Comment:
Ok, we can't use rsync for this merge unless we also replace build kite

Reply:
And we're using rsync so that things don't explode when you run yarn build twice and the files already exist (which excludes `mv` without some extra steps to remove them and not also fail when the files aren't there because someone ran it for the first time)

Reply:
I don't quite understand; are you saying `mv` will error if you try to copy a file to a destination that already exists?

Reply:
Specifically directories. If the directory already exists then it'll complain like e.g.``mv: cannot move 'lib/src/report' to 'lib/report': Directory not empty`.

Reply:
does `cp -r` do the same

Reply:
oh rip :face_with_spiral_eyes: